### PR TITLE
fix: Correct asset name in release pipeline

### PR DESCRIPTION
**Pull request by tianfeng92**
# One-line summary

The asset name in release pipeline is wrong. Use `playwright` instead of `testcafe`. 🤷‍♀️

### CODE REVIEW

**Review by mhan83 — COMMENTED** [review]
Yikes. So is this change going to break the release workflow though? A lot of the steps after parsing the ASSET_ID has conditions like:
```
- name: Checkout
  uses: actions/checkout@v3
  if: ${{ steps.prep.outputs.asset_id == '' }}
  with:
    ref: ${{ steps.prep.outputs.version }}
```

**Comment by tianfeng92** [comment]
> Yikes. So is this change going to break the release workflow though? A lot of the steps after parsing the ASSET_ID has conditions like:
> 
> ```
> - name: Checkout
>   uses: actions/checkout@v3
>   if: ${{ steps.prep.outputs.asset_id == '' }}
>   with:
>     ref: ${{ steps.prep.outputs.version }}
> ```

I checked the release pipelines, the original PR, as well as other runners' pipeline. I don't find any `asset_id` in the API and it's always empty whether it's official release or beta release. Also I can't really recall why it's introduced and what it wants to solve. I'm going to discuss it in standup and see if we can remove this check.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -182,7 +182,7 @@ jobs:
 
           ASSET_ID=$(curl -s -H "Authorization: token ${{ github.token }}" \
                       https://api.github.com/repos/${{ github.repository }}/releases | \
-                      jq -r "[.[] | select(.body | contains(\"- jobId: ${{ github.run_id }}\\n\"))] | first | .assets | .[] | select(.name == \"sauce-testcafe-macos.zip\") | .id | select(. != null)")
+                      jq -r "[.[] | select(.body | contains(\"- jobId: ${{ github.run_id }}\\n\"))] | first | .assets | .[] | select(.name == \"sauce-playwright-macos.zip\") | .id | select(. != null)")
```
Comment (alexplischke) [inline .github/workflows/release.yml:185]: Nice find! But the assets are called [playwright-macos-amd64.zip](https://github.com/saucelabs/sauce-playwright-runner/releases/download/v2.25.0/playwright-macos-amd64.zip) and [playwright-windows-amd64.zip](https://github.com/saucelabs/sauce-playwright-runner/releases/download/v2.25.0/playwright-windows-amd64.zip) respectively.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -81,6 +81,7 @@ jobs:
     needs: [create-release-draft]
     steps:
       - name: Find Matching Draft Tag
+        description: Retrieves the `asset_id` of the uploaded bundle. An non-empty `asset_id` indicates that the asset has been successfully uploaded, thus avoiding redundant uploads of the bundle on subsequent retries of this step.
```
Comment (alexplischke) [inline .github/workflows/release.yml:84]:
```suggestion
        description: Retrieves the `asset_id` of the uploaded bundle. A non-empty `asset_id` indicates that the asset has been successfully uploaded, thus avoiding redundant uploads of the bundle on subsequent retries of this step.
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -166,6 +167,7 @@ jobs:
...
```
Comment (alexplischke) [inline .github/workflows/release.yml:170]:
```suggestion
        description: Retrieves the `asset_id` of the uploaded bundle. A non-empty `asset_id` indicates that the asset has been successfully uploaded, thus avoiding redundant uploads of the bundle on subsequent retries of this step.
```

File: .github/workflows/release.yml (hunk 2 above)
Comment (tianfeng92) [reply .github/workflows/release.yml]: I refined it in another edition. 😄

**Review by alexplischke — APPROVED** [review]

**Pull request merged**